### ISSUE TITLE
New recipe: Mining of electronic health records (EHR) using natural language processing (NLP)

### ISSUE THREAD

**Issue by YojanaGadiya** [FAIRplus/the-fair-cookbook#378]
If you needed to add a task to the list below, please think about amending the issue template: https://github.com/FAIRplus/the-fair-cookbook/blob/migrating/.github/ISSUE_TEMPLATE/meta-checklist.md
Great! Now, to the actual tasks:

- [x] identify author
- [x] write abstract
- [ ] agree with editors on abstract
- [ ] write recipe
- [ ] identify reviewer
- [ ] conduct review
- [ ] incorporate reviewer's comments
- [ ] publish recipe

**Comment by YojanaGadiya** [comment]
@robertgiessmann We have a branch which @proccaserra and I are working on. Would you want us to link this issue to that branch directly for the abstract or you would like me to post the abstract here itself?

**Comment by robertgiessmann** [comment]
hi @YojanaGadiya , it would be the nlp2rdf branch, then, wouldn't it?

Shall we treat this below as the abstract? 


> Despite the progress in organizing scientific knowledge in specialized databases as evidenced by the ever expanding number of resources available from 
> [FAIRsharing.org](https://www.fairsharing.org), unstructured text remains in many areas how domain knowledge is held in a field. 
> `Unstructured text` refers simply to natural language text as found in journal articles, scientific reports or medical notes. The qualifier `unstructured` refers to the fact that the information is available simply as a sequence of words, without any markup or annotation.
> While these documents are meaningful to humans, it presents two challenges. To the humans, reading and synthesizing the volumes of texts available is impossible. To computers and software agents, it was, until recently, extremely difficult to **extract meaning** from **unstructured text.**
> However, in recent years, the fields of computational linguistics and machine learning have seen breakthroughs in 'natural language processing', which enable researchers and scientists to tap into the knowledge mines that constitutes unstructured text.  Tools such as `BERT`, short for Bidirectional Entity Recognition Transformers,  now known collectively as `Transformers` have transformed our ability to exploit unstructured text.
> The tasks of Named Entity Recognition (NER) and Relation Recognition, critical to accomplishing 'natural language understanding' (NLU) can now be done with higher confidence and, as evidenced by the increasing number of publications, is delivering advances in biomedicine and healthcase which benefit to patients and science advancement.

**Comment by YojanaGadiya** [comment]
Yes, it would be the nlp2rdf branch. I think we can treat the above part as abstract. @proccaserra do you agree on this?

**Comment by proccaserra** [comment]
@robertgiessmann @YojanaGadiya that's indeed the starting point.
AZ, Novartis and other EFPIAs have pipelines in place (see links provided in the outline). The idea is to provide content detailing the key steps.

**Comment by robertgiessmann** [comment]
felt there's not much more I can do, so removed my assignment
